source/features/vertical-front-matter.css

```
@@ -1,3 +1,3 @@
-.rgh-front-matter-name {
+.rgh-vertical-front-matter-table th {
```

i would use the `tbody` if there later is a header, this gets confused
```suggestion
.rgh-vertical-front-matter-table tbody tr th {
```